hieroplatform/src/test/java/org/hiero/sketch/IntConverter.java

```diff
@@ -0,0 +1,13 @@
+package org.hiero.sketch;
```

Comment:
The converter is only needed for String columns, where it is mandatory.
For other data types we already have built-in converters.